pkg/plugins/plugins.go

```diff
@@ -252,20 +265,45 @@ func (scanner *PluginScanner) loadPluginJson(pluginJsonFilePath string) error {
 	}
 
 	if pluginCommon.Id == "" || pluginCommon.Type == "" {
-		return errors.New("Did not find type and id property in plugin.json")
+		return errors.New("did not find type or id properties in plugin.json")
+	}
+
+	pluginCommon.PluginDir = filepath.Dir(pluginJsonFilePath)
```

Comment:
This puts MANIFEST.txt in the src/ directory of the plugin, should it be in the root directory instead?

Reply:
You usually don't have a src directory in plugin when it's published - then it consists of built JavaScript, binaries etc in the root together with manifest. 

Maybe easier to reuse existing plugin example and mock the part that says if it's signed or unsigned?

Reply:
Another idea would be to verify signed/unsigned in a later step, like [here](https://github.com/grafana/grafana/blob/master/pkg/plugins/models.go#L73) or similar as [here](https://github.com/grafana/grafana/blob/dda8caf48c264c1f309b5d1f25e199095b4bf0d1/pkg/plugins/plugins.go#L108)

Reply:
@marefr Ah thanks, I can get rid of the src/ directory then.